CONTRIBUTING.md

```diff
@@ -0,0 +1,99 @@
+# Contributing to KoLmafia
+
+First off, thank you for contributing your time and knowledge. This project
+would not exist if not for volunteers like you!
+
+## Some useful prerequisites (lifted from [Github docs](https://github.com/github/docs/blob/main/CONTRIBUTING.md))
+
```

Comment:
Done. I renamed the section, and added a comment regarding "in the spirit of github flow, we do things this way" Let me know if that's any better!

Reply:
Do we want to link to the (not currently up to date) build and IDE setup guides on the wiki?  Should there be general links to the wiki in the suggested reading section?

Reply:
I guess my question on this front is "what do we see as the purpose of the wiki?" I view it primarily as a user guide, not as a developer guide. It could be both, but I think it makes sense to put dev-only content where devs are already looking.

Reply:
If someone needs help in setting things up and we want to preserve the interaction so we don't have to repeat ourselves for some visitor from the future, I think there should be some minimal guidance on KoLmafia.us.  We do have at least one person who wants to setup and build locally but has no desire to share what they are doing so that conversation might fit better there then here.  If we can clearly identify something of benefit/interest only to devs then it belongs here.

Reply:
Hey, if you've already downloaded the repo, you'll have a copy of CONTRIBUTING.md locally :)